www/c-icap/src/opnsense/mvc/app/models/OPNsense/CICAP/General.xml

```diff
@@ -0,0 +1,59 @@
+<model>
+    <mount>//OPNsense/cicap/general</mount>
+    <description>c-icap configuration</description>
+    <version>1.0.0</version>
+    <items>
+        <enabled type="BooleanField">
+            <default>0</default>
+            <Required>Y</Required>
+        </enabled>
+        <timeout type="IntegerField">
+            <default>300</default>
+            <Required>Y</Required>
+        </timeout>
+        <maxkeepaliverequests type="IntegerField">
+            <default>100</default>
+            <Required>Y</Required>
+        </maxkeepaliverequests>
+        <keepalivetimeout type="IntegerField">
+            <default>600</default>
+            <Required>Y</Required>
+        </keepalivetimeout>
+        <startservers type="IntegerField">
+            <default>3</default>
+            <Required>Y</Required>
+        </startservers>
+        <maxservers type="IntegerField">
+            <default>10</default>
+            <Required>Y</Required>
+        </maxservers>
+        <minsparethreads type="IntegerField">
+            <default>10</default>
+            <Required>Y</Required>
+        </minsparethreads>
+        <maxsparethreads type="IntegerField">
+            <default>20</default>
+	    <Required>Y</Required>
+	</maxsparethreads>
+        <threadsperchild type="IntegerField">
+            <default>10</default>
+	    <Required>Y</Required>
+	</threadsperchild>
+        <maxrequestsperchild type="IntegerField">
+            <default>0</default>
+	    <Required>Y</Required>
+	</maxrequestsperchild>
+        <listenaddress type="TextField">
+            <default>127.0.0.1</default>
+            <Required>Y</Required>
+        </listenaddress>
+        <serveradmin type="TextField">
+            <default>you@your.address</default>
+            <Required>Y</Required>
+        </serveradmin>
+        <servername type="TextField">
+            <default>YourServerName</default>
+            <Required>Y</Required>
+        </servername>
```

Comment:
Yes - the PHP stuff and jinja a not connected together. All jinja functions, helpers etc. are globally usable. The only thing is, that the legacy pages usually somewhere in the root object in a direct child of it so there is no package name prefix like in the MVC apps.

This is why the new static routes model looks a bit different in the mount point etc. -> it reuses this old path instead of a package like it's usually done by the MVC apps (it has no template).